src/content/en/updates/2017/04/native-mac-os-notifications.md

```diff
@@ -0,0 +1,145 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: As of Chrome version 59, notifications will be displayed native notifications on Mac OS X.
```

Comment:
Remove 'version'. 'Chrome 59' is sufficient.

Also, I think 'displayed native notifications' should be 'displayed as native notifications'.